packages/ai-jsx/src/stream/index.ts

```diff
@@ -99,6 +99,27 @@ export function toStreamResponse(renderable: Renderable): Response {
   );
 }
 
+/**
+ * Converts a {@link Renderable} to a {@link ReadableStream} that will stream the rendered
+ * content as UTF-8 encoded text.
```

Comment:
If this is user-facing, it would be good to add a sentence about the tradeoffs of doing this versus other approaches, and what those alternatives are.

Reply:
Good call